mako_config/README.md

```diff
@@ -0,0 +1,4 @@
+This directory collects all Mako related files. The `uploader` is located at
+[iree-mako](https://github.com/hanhanW/iree-mako/tree/iree). The Mako repo is
```

Comment:
Hmm having a personal fork for doing this is a bit gross...

Reply:
yeah, I was testing it with my personal repo. Any idea where to put it?

Reply:
I guess we should probably create google/iree-mako if we really want a separate repo. IREE should not depend on your personal repo in any long-term capacity.

Reply:
SG, I think we can leave it as this way for now. I will figure out how to create google/iree-mako (or ask someone to help) and we can replace it.